aggregators/can-load-offline/index.js

```diff
@@ -19,14 +19,25 @@
 
 const Aggregate = require('../aggregate');
 
+/**
+ * @type {string}
+ */
+const serviceWorker = require('../../audits/offline/service-worker').name;
```

Comment:
hmm, do these not get figured out automatically? Closure should be able to figure out the type itself